config/src/main/java/com/typesafe/config/impl/Parseable.java

```diff
@@ -180,6 +180,8 @@ final private AbstractConfigValue parseValue(ConfigOrigin origin,
             return rawParseValue(origin, finalOptions);
         } catch (IOException e) {
             if (finalOptions.getAllowMissing()) {
+                trace(e.getMessage() + ". Allowing Missing File, this can be turned off by setting" +
+                        " ConfigParseOptions.allowMissing = false");
```

Comment:
Does the filename end up in this message somehow? (not sure if it's in getMessage or added by trace or what)

Reply:
This is one of the messages from trace:
`resource not found on classpath: nothere.json. Allowing Missing File, this can be turned off by setting ConfigParseOptions.`

So it does.

Reply:
great! thanks again